src/main/java/mate/academy/hibernate/relations/service/impl/ActorServiceImpl.java

```diff
@@ -1,16 +1,30 @@
 package mate.academy.hibernate.relations.service.impl;
 
+import java.util.Optional;
+import mate.academy.hibernate.relations.dao.ActorDao;
+import mate.academy.hibernate.relations.dao.impl.ActorDaoImpl;
+import mate.academy.hibernate.relations.exception.DataProcessingException;
 import mate.academy.hibernate.relations.model.Actor;
 import mate.academy.hibernate.relations.service.ActorService;
 
 public class ActorServiceImpl implements ActorService {
+    private final ActorDao abstractDao;
+
+    public ActorServiceImpl(ActorDaoImpl actorDao) {
+        this.abstractDao = actorDao;
+    }
+
     @Override
     public Actor add(Actor actor) {
-        return null;
+        return abstractDao.add(actor);
     }
 
     @Override
     public Actor get(Long id) {
-        return null;
+        Optional<Actor> movie = abstractDao.get(id);
```

Comment:
The variable name `movie` is misleading in this context. It should be renamed to something like `actor` to reflect that it holds an `Actor` object.